scripts/less-watch

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+/**
+
+Usage:
+./scripts/less-watch [watchPath]
+
+Example:
+./scripts/less-watch ./notebook/static/notebook/less
+
+**/
+
+var less = require('less');
+var fs = require('fs');
+var path = require('path');
+var child_process = require('child_process');
+
+function watchDir(dir) {
+  var rootPath = path.join(__dirname, '..');
+  var watchPath = path.resolve(dir);
+  console.log('less-watch:', 'watching:', path.relative(rootPath, watchPath));
+  fs.watch(watchPath, {recursive: true}, function(event, file) {
+    if (file && /.+\.less$/.test(file)) {
+      console.log('less-watch:', 'modified:', file);
+      child_process.exec('lessc --include-path=notebook/static --verbose notebook/static/style/style.less notebook/static/style/style.min.css', function(err, stdout, stderr) {
```

Comment:
I got `npm run build:css` working from this script, but the issue is when you break the LESS compilation (e.g. commenting out the Bootstrap import in style.less), the error message looks like:

```
[0] less-watch: modified: style/style.less
[0] { [Error: Command failed: npm run build:css
[0] 
[0] npm ERR! Darwin 15.6.0
[0] npm ERR! argv "/usr/local/bin/iojs" "/usr/local/bin/npm" "run" "build:css"
[0] npm ERR! node v5.12.0
[0] npm ERR! npm  v3.8.6
[0] npm ERR! code ELIFECYCLE
[0] npm ERR! jupyter-notebook-deps@4.0.0 build:css: `concurrent "npm run build:css:ipython" "npm run build:css:style"`
[0] npm ERR! Exit status 1
[0] npm ERR! 
[0] npm ERR! Failed at the jupyter-notebook-deps@4.0.0 build:css script 'concurrent "npm run build:css:ipython" "npm run build:css:style"'.
[0] npm ERR! Make sure you have the latest version of node.js and npm installed.
[0] npm ERR! If you do, this is most likely a problem with the jupyter-notebook-deps package,
[0] npm ERR! not with npm itself.
[0] npm ERR! Tell the author that this fails on your system:
[0] npm ERR!     concurrent "npm run build:css:ipython" "npm run build:css:style"
[0] npm ERR! You can get information on how to open an issue for this project with:
[0] npm ERR!     npm bugs jupyter-notebook-deps
[0] npm ERR! Or if that isn't available, you can get their info via:
[0] npm ERR!     npm owner ls jupyter-notebook-deps
[0] npm ERR! There is likely additional logging output above.
[0] 
[0] npm ERR! Please include the following file with any support request:
[0] npm ERR!     /Users/grant/Sites/jupyter/notebook/npm-debug.log
[0] ] killed: false, code: 1, signal: null, cmd: 'npm run build:css' }
```

whereas the current error message looks like:

```
[0] less-watch: modified: style/style.less
[0] { [Error: Command failed: lessc --include-path=notebook/static --verbose notebook/static/style/style.less notebook/static/style/style.min.css
[0] NameError: variable @body-bg is undefined in /Users/grant/Sites/jupyter/notebook/notebook/static/notebook/less/variables.less on line 4, column 40:
[0] 3 
[0] 4 @cell_selected_background:      darken(@body-bg, 2%);
[0] 5 @cell_background:               darken(@body-bg, 3.2%);
[0] 
[0] ]
[0]   killed: false,
[0]   code: 1,
[0]   signal: null,
[0]   cmd: 'lessc --include-path=notebook/static --verbose notebook/static/style/style.less notebook/static/style/style.min.css' }
```